### PR TITLE
M24Y-2/sass partials

### PR DESCRIPTION
**Pull request by kimntan**
Added breakpoints, colors, font-size, fonts, mixins, and spacing partials. 
Included button and label mixins.

### CODE REVIEW

File: src/styles/partials/_spacing.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@use './breakpoints' as *;
+
+@mixin padding {
+  padding: 0 1rem;
+  @include tablet {
+    padding: 0 2rem;
+  }
```
Comment (tina96pham) [inline src/styles/partials/_spacing.scss:7]: @mixin global-padding{
    display: flex;
    gap: 1rem;
    padding-left: 1rem;
    padding-right: 1rem;
    box-sizing: border-box;

    @include tablet {
        padding-left: 1.5rem;
        padding-right: 1.5rem;
        gap: 1.5rem;
    }

    @include desktop{
        padding:0 calc((100% - 64.5rem) * 0.5);
    }
}

File: src/styles/partials/_spacing.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@use './breakpoints' as *;
+
+@mixin padding {
+  padding: 0 1rem;
```
Comment (tina96pham) [inline src/styles/partials/_spacing.scss:4]: display: flex;
    flex-wrap: wrap;
    box-sizing: border-box;

File: src/styles/partials/_font-size.scss
Comment (tina96pham) [inline src/styles/partials/_font-size.scss:1]: Should we create a _global.css and put all the fon't sizes under the heading and paragraph elements tag so we don't have to type it down every time we style? We just need to import to the components scss and we good to go.

Comment (kimntan) [reply src/styles/partials/_font-size.scss:1]: I usually include all of my global styles in the app.scss file, so I'd also set margin and padding of the body to 0, and do box-sizing as well. Then I'd set all my h1, h2. h3 using the font-size mixins.

File: src/styles/partials/_spacing.scss (hunk 2 above)
Comment (kimntan) [reply src/styles/partials/_spacing.scss:4]: What is the display flex for? This mixin is also in a spacing partial, so I don't know if I'd include the display property here. Maybe in the mixins partial, which includes all other mixins unrelated to spacing.

Comment (tina96pham) [reply src/styles/partials/_spacing.scss:4]: I was looking at setting genral padding as create container so having display flex but less work on importing it

**Comment by crystdang** [comment]
looks good!

**Review by crystdang — APPROVED** [review]

**Pull request merged**